app/controllers/restaurants_controller.rb

```diff
@@ -0,0 +1,45 @@
+class RestaurantsController < ApplicationController
+   def index
+     @restaurant = Restaurant.all
+   end
+ 
+   def show
+     @restaurant = Restaurant.find(params[:id])
+   end
+ 
+   def new
+     @restaurant = Restaurant.new
+   end
+ 
+   def edit
+     @restaurant = Restaurant.find(params[:id])
+   end
+ 
+   def create
+     @restaurant = Restaurant.create(restaurant_params)
+     if @restaurant.save 
+       redirect_to root_path
+       flash[:notice] = "Restaurant  was  saved  successfully"
+     else 

```

Comment:
@joni88 you're right, this is my mistake I am confusing this with the other application :)

Yes it should be restaurants_path

```
def create
  @restaurant = Restaurant.create(restaurant_params)
  redirect_to restaurants_path 
end
```